recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
+
+source:
+    git_url: https://github.com/PivotalRD/libhdfs3.git
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
```

Comment:
I added a similar comment to `hdfs3`, `libgsasl`, and `libntlm` per discussion in those recipes as to why it was Linux-only.